code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -183,6 +183,10 @@ var/global/list/image/splatter_cache=list()
 /obj/effect/decal/cleanable/blood/gibs/core
 	random_icon_states = list("gibmid1", "gibmid2", "gibmid3")
 
+/obj/effect/decal/cleanable/blood/gibs/New()
```

Comment:
I'd personally recommend adding noclear = 1 to gibs so that the effect remains, even after you scoop it into a beaker.